src/lib/__tests__/e2e/GeneralSvgRenderingTest.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import { SvgFromXml } from 'react-native-svg';
```

Comment:
I think that `__tests__` folder should not be inside `src` folder (or at least it should be excluded from bundle).

Reply:
I moved that file to `__tests_/e2e` directory